test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/AssemblyAndClassInitializeTimeout.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Testing.Platform.Acceptance.IntegrationTests.Helpers;
```

Comment:
Today this project contains only acceptance of the platform not MSTest. I am fine to start adding tests there but it'd be nice to rename the project into `MSTest.Acceptance.IntegrationTests`. and add a folder (no need to update namespace) for MSTest related tests and another for platform.

Reply:
done